### ISSUE TITLE
[Bug] Unable to create SSD volume on Flexible Engine

### ISSUE THREAD

**Issue by wruffine-csgroup** [CS-SI/SafeScale#115]
## Version used
Safescaled version: 21.11.0, build 62f078347ba043e3c88f4ab2af7672a911b655ff (2021/12/23-16:02), with Tags: (release,)

## Bug description

Volume creation fails for SSD volumes on Flexible Engine:
safescale logs:
```
safescale volume create --speed SSD test                                       
{"error":{"exitcode":6,"message":"Cannot create volume: stopping retries: {\"itemNotFound\": {\"message\": \"Volume type with name Ssd could not be found.\", \"code\": 404}}"},"result":null,"status":"failure"}
```

safescaled logs:
```
ERROR   [2022-02-04 11:28:49.369][error][   4439] listeners.(*VolumeListener).Create() [.../go/src/github.com/CS-SI/SafeScale/lib/server/listeners/volume.go:138]: stopping retries: {"itemNotFound": {"message": "Volume type with name Ssd could not be found.", "code": 404}}
```
HDD and COLD volume creation works flawlessly.

**Comment by oscarpicas** [comment]
Fixed by 25d509b0a11758668e464fd0b05b37ac1ba7aaed

**Comment by oscarpicas** [comment]
In order to test the fix, you should build the branch 'develop' or wait for next release.
Assuming you have git, and docker in your machine, the fastest way to build is:

```
git clone https://github.com/CS-SI/SafeScale -depth 1 -b develop
cd SafeScale
./create-local-docker.sh

```
After that, you should see a ./exported directory, in there you should found the safescale and safescaled binearies
